cmd/ocm-backplane/login/login.go

```diff
@@ -73,6 +74,13 @@ func init() {
 		"Save kube configuration in the specific path when login to multi clusters.",
 	)
 
+	flags.StringVar(
+		&args.defaultNamespace,
+		"defaultns",
+		"openshift-backplane-srep",
+		"Sets a default namespace for SRE to run commands like oc debug node/xxx",
```

Comment:
Thank you.  I have renamed in this
https://github.com/openshift/backplane-cli/pull/315/commits/661d5d4399befc5709c0b93c62b4a8aa81108e49

> Quick Small change - Can we rename the flag from defaultns to `--namespace` So that it stays a bit consistent with oc We can also add single letter flag `-n`


Thank you.  I have renamed and added a single letter flag too in this
https://github.com/openshift/backplane-cli/pull/315/commits/661d5d4399befc5709c0b93c62b4a8aa81108e49